Sources/iOS/OAuth2Authorizer+iOS.swift

```diff
@@ -142,9 +145,10 @@ open class OAuth2Authorizer: OAuth2AuthorizerUI {
 					self.oauth2.logger?.warn("OAuth2", msg: "Cannot intercept redirect URL: \(err)")
 				}
 			} else {
-				self.oauth2.didFail(with: nil)
+				self.oauth2.didFail(with: error?.asOAuth2Error)
```

Comment:
I think that the `oauth2` depends on this to be `nil` in order to process and provide `OAuth2Error.requestCancelled` error.
Please see `Oauth2Base.didFail(with:)` method for details.